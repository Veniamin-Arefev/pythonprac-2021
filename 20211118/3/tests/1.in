alp = Alpha(c=10, z=2, a=42)
alp.e = 123
print(alp)