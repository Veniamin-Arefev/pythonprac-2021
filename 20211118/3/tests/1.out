<class 'AttributeError'>
q
<class 'AttributeError'>

